addons/hr_homeworking/wizard/calendar_popover_delete_wizard_homework.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class CalendarPopoverDeleteWizardHomework(models.TransientModel):
+    _name = 'calendar.popover.delete.wizard.homework'
+    _description = 'Calendar Popover Delete Wizard Homework'
+
+
+    record = fields.Many2one('hr.homeworking')
+    date = fields.Date()
+    delete = fields.Selection([('one', 'Delete only this day\'s work location'), ('all', 'Delete this work location for everyweek')], default='one')
```

Comment:
Seems that all the fields should be required. (+ avoid escaping in labels for readability purposes)
```suggestion
    delete = fields.Selection([
        ('one', "Delete only this day's work location"),
        ('all', "Delete this work location for everyweek")
    ], default='one', required=True)
```